espnet2/iterators/chunk_iter_factory.py

```diff
@@ -86,6 +88,11 @@ def __init__(
         self.batch_size = batch_size
         self.seed = seed
         self.shuffle = shuffle
+        self.excluded_key_pattern = (
+            "(" + "[0-9]*)|(".join(excluded_key_prefixes) + "[0-9]*)"
```

Comment:
Are you only assuming the cases of numbers are appended?
If so, it should be documented in `add_argument` or other places.
Adding a comment to the config is also informative.